scripts/jenkins-yoctobuild-build.sh

```diff
@@ -180,8 +141,8 @@ copy_build_conf() {
         local src="$1"
         local tmpfile="$(mktemp)"
 
-        if grep /home/jenkins "$src"; then
-            echo "Please do not specify /home/jenkins directly in any build-conf files. Use @WORKSPACE@."
+        if grep /home/mender "$src"; then
+            echo "Please do not specify /home/mender directly in any build-conf files. Use @WORKSPACE@."
```

Comment:
This is not correct, AFAIK. I think Gitlab uses `/build/Northern.tech` or something like that, doesn't it?

Strictly speaking you could probably remove this. It was from a time when we were still debugging a lot on the runners themselves, and paths often got hardcoded. I think that's unlikely today.

Reply:
You are right. And I agree with just removing it.